apps/dashboard/src/utils/beforeLoadUtil.ts

```diff
@@ -5,8 +5,15 @@ import apiService from '@/services/ApiService';
 export default async function beforeLoad() {
 
   const settingsStore = useSettingsStore();
+  try {
   await settingsStore.fetchMaintenanceMode();
   await settingsStore.fetchKeys();
```

Comment:
A bit nit-picky but it might be nice to indent these since they are in a try block now